qemu/tests/x86_cpu_flag_intel_pt.py

```diff
@@ -0,0 +1,63 @@
+import time
+import logging
+
+from avocado.utils import process
+from virttest import error_context, env_process
+from provider.cpu_utils import check_cpu_flags
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test cpu flag intel-pt.
+    1) Check if current flags are in the supported lists, if no, cancel test
+    2) Otherwise, set pt_mode = 1 on host at first
+    3) Boot guest with cpu model 'host' without intel-pt.
+    4) Check cpu flags in guest(only for linux guest)
+    5) For q35
+       5.1) boot guest with cpu model with intel-pt
+    6) For pc
+       6.1) boot guest with intel-pt and min-level=0x14
```

Comment:
Seems step 3 is after step 6 actually.

Reply:
Seems you boot up guest twice in this case, the only difference is the param cpu model and flag. It can be implemented by add variants into cfg and split to two cases. I'm not sure which one is better. @vivianQizhu What do you think?